src/ipc/dbus/ipc-dbus.h

```diff
@@ -1038,7 +1040,7 @@ class InterfacePropertyIPCAdapterHandler : public InterfacePropertyHandlerBase
     }
 
     QPointer<InterfaceType> m_service;
-    InterfaceAdapterType* m_serviceAdapter = nullptr;
+    QPointer<InterfaceAdapterType> m_serviceAdapter;
```

Comment:
QPointer is not for free. objectPath() cannot be called with a dangling pointer, since only InterfacePropertyIPCAdapterHandler destructor will destruct m_serviceAdapter.

Reply:
InterfacePropertyIPCAdapterHandler is not the parent of the object stored in m_serviceAdapter